### PR TITLE
fix: showing download confirmation snackbar message after saving assets [AR-1729]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1729" title="AR-1729" target="_blank">BugAR-1729</a>  [download/save assets] "success" toast is sometimes shown behind the dialog
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
After downloading some asset correctly, the toast message was being shown only once because we were clearing the state view on the wrong VM. With this fix, after displaying a message, the snackbar messages view states values will be cleared on both `MessageComposerViewModel` and `ConversationMessagesViewModel`

#### How to Test

- Receive/Send an asset. 
- Click on Save asset.
- See how the snackbar message is displayed.
- Repeat the process and check the toast appears again.

### Attachments (Optional)
https://user-images.githubusercontent.com/2468164/209695082-6b8905db-7e32-4a5b-a40d-7573946f640c.mp4 https://user-images.githubusercontent.com/2468164/209695110-11b826c2-53f3-41ec-9105-f2ae94df0c81.mp4

----

#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1265/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5319192)(5319192)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5319192)(5319192)-dev-compat.apk)

**Review by saleniuk — COMMENTED** [review]
Would it be difficult to change it to already use `Flow` like in `OtherUserProfileScreenViewModel` - the way we agreed some time ago? 🤔 We wouldn't have to worry about cleaning up the state after showing the snackbar

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1265/2/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt [hunk 1]
```diff
@@ -474,57 +468,36 @@ private fun ConversationScreenContent(
         membersToMention = membersToMention
     )
 
-    SnackBarMessage(snackbarMessage, conversationState, conversationScreenState, onSnackbarMessageShown)
+    SnackBarMessage(composerMessages, conversationMessages, conversationScreenState)
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt]: I think this `SnackBarMessage` could be used in `ConversationScreen` where `conversationScreenState` is created and we wouldn't have to pass both Flows all the way to the `ConversationScreenContent`

**Review by saleniuk — APPROVED** [review]
Looks good, just one little suggestion, but it's not necessary. Great that you were able to refactor it 😄

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1265/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5327323)(5327323)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5327323)(5327323)-dev-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt (hunk 1 above)
Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt]: Thanks and done.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1265/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5327450)(5327450)-dev-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/compat/com.wire.android-vv0.0.1(5327450)(5327450)-dev-compat.apk)

**Pull request merged**